src/App.js

```diff
@@ -1,36 +1,39 @@
 import { Switch, Route, Redirect } from 'react-router-dom';
 import { PokedexView } from './components/PokedexView';
+import PokemonsProvider from './context/PokemonsProvider';
 import { withRouter } from './HOCs';
-import generations from './generations';
+import generations from './data/generations';
 import './App.css';
 
 function App() {
 	return (
-		<div className="pokedex-app">
+		<PokemonsProvider>
+			<div className="pokedex-app">
 
-			<h1>
+				<h1>
 				P<i></i>kédex
-			</h1>
+				</h1>
 
-			<Switch>
+				<Switch>
 
-				<Route exact path="/">
-					<Redirect to={ generations[ 0 ].link } />
-				</Route>
+					<Route exact path="/">
+						<Redirect to={ generations[ 0 ].link } />
+					</Route>
 
-				{
-					generations.map( ( gen, i ) => (
+					{
+						generations.map( ( gen, i ) => (
 
-						<Route key={ i } exact path={ '/' + gen.link }>
-							<PokedexView generation={ gen } />
-						</Route>
+							<Route key={ i } exact path={ '/' + gen.link }>
```

Comment:
Don't use the index as a key, use a value from the gen.

Reply:
Done